ui/components/DataTable/TableView/TableView.tsx

```diff
@@ -0,0 +1,57 @@
+import { Table, TableContainer } from "@material-ui/core";
+import React from "react";
+import TableBodyView from "./TableBody";
+import TableHeader from "./TableHeader";
+import { TableViewProps } from "./types";
+
+const TableView = ({
+  rows,
+  fields,
+  id,
+  checkedFields,
+  defaultSortedField,
+  onSortChange,
+  hasCheckboxes,
+  onBatchCheck,
+}: TableViewProps) => {
+  const onCheckChange = (checked: boolean, id: string) => {
+    const chk = [...checkedFields];
```

Comment:
💡 I think we could use a more informative variable name here (preferably in plural) — to reflect that this variable holds an array of strings.

Reply:
i'll rename it to `selectedFields` :ok: